### ISSUE TITLE
Add batch norm to the convolution layers

### ISSUE THREAD

**Issue by mfranzs** [MIT-6819-team/TF_colorization#10]

**Comment by kocabey** [comment]
Check out https://www.tensorflow.org/versions/r0.9/api_docs/python/nn.html#batch_normalization

**Comment by mfranzs** [comment]
gj enes
